### DISCUSSION TITLE
What is the Difference between Tailwind CSS and Bootstrap?

### DISCUSSION THREAD

**Discussion by koj4** [community/community#147417] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm deciding on a CSS framework to use in a web development project. What are the main differences between Tailwind CSS and Bootstrap, and when is it best to use one of the two?

**Comment by hkqr** [comment]


**Reply by koj4** [reply]
Thanks a lot! I don't know what would happen without your help.

**Reply by hkqr** [reply]
ok
